extensions/smallrye-fault-tolerance/deployment/src/main/java/io/quarkus/smallrye/faulttolerance/deployment/FaultToleranceMethodSearch.java

```diff
@@ -0,0 +1,408 @@
+package io.quarkus.smallrye.faulttolerance.deployment;
+
+import java.lang.reflect.Modifier;
+import java.util.ArrayDeque;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Deque;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.jboss.jandex.ClassInfo;
+import org.jboss.jandex.ClassType;
+import org.jboss.jandex.DotName;
+import org.jboss.jandex.IndexView;
+import org.jboss.jandex.MethodInfo;
+import org.jboss.jandex.ParameterizedType;
+import org.jboss.jandex.Type;
+import org.jboss.jandex.TypeVariable;
+import org.jboss.jandex.VoidType;
+import org.jboss.jandex.WildcardType;
+
+import io.quarkus.arc.processor.AssignabilityCheck;
+import io.quarkus.arc.processor.KotlinDotNames;
+
+// copy of `io.smallrye.faulttolerance.config.SecurityActions` and translation from reflection to Jandex
```

Comment:
Don't we need to include the copyright notice from https://github.com/smallrye/smallrye-fault-tolerance/blob/d85d79e02ed704945dce1ddf1dae38834733e89e/implementation/fault-tolerance/src/main/java/io/smallrye/faulttolerance/config/SecurityActions.java#L1-L15 as discussed in https://quarkusio.zulipchat.com/#narrow/stream/187038-dev/topic/Reusing.20code.20from.20different.20project.20under.20apache.20license?

Reply:
That file is authored by myself and @michalszynkiewicz in the past, when he was at Red Hat, so I didn't think this matters, but I'll add a copy of the header here just to be sure.

Reply:
Done. Thanks!